index.bs

```diff
@@ -6754,6 +6754,14 @@ Note: this extension may be implemented for [=authenticators=] that do not use [
     dictionary AuthenticationExtensionsPRFInputs {
         AuthenticationExtensionsPRFValues eval;
         record<USVString, AuthenticationExtensionsPRFValues> evalByCredential;
+        AuthenticationExtensionsPRFImportCryptoKeyParams importCryptoKey;
+    };
+
+    dictionary AuthenticationExtensionsPRFImportCryptoKeyParams {
+        required KeyFormat format;
```

Comment:
Nothing but [Raw](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/importKey#raw) can work as a format, right?

Reply:
Right, at least not without preprocessing by the client.

For example, I did one experiment where my application code wrapped the 32 raw bytes in a PKCS#8 header and fed it into `importKey` as a P-256 private key. It "works" (except when it doesn't because of the key domain mismatch, see other thread below), but it's quite an ugly hack.